docs/tempo/website/grafana-agent/_index.md

```diff
@@ -0,0 +1,126 @@
+Traces that arrive to a Tempo instance will enter the same tracing 
+pipeline, and will be processed equally.
+
+### Pipeline processing
+
+The Grafana Agent processes tracing data as it flows through the pipeline to make the distributed tracing system more reliable and leverage the data for other purposes.
+Such as trace discovery, tail-based sampling, and generating metrics.
+
+#### Batch
+
+The Agent supports batching of traces.
+Batching helps better compress the data and reduce the number of outgoing connections required to transmit the data.
+To configure it, refer to the `batch` block in the [config reference](https://github.com/grafana/agent/blob/main/docs/configuration-reference.md#tempo_instance_config).
+
+#### Attributes manipulation
+
+The Grafana Agent allows for general manipulation of tags on spans that pass through this agent.
+A common use may be to add an environment or cluster variable.
+To configure it, refer to the `attributes` block in the [config reference](https://github.com/grafana/agent/blob/main/docs/configuration-reference.md#tempo_instance_config).
+
+#### Attaching metadata with Prometheus Service Discovery
+
+Prometheus Service Discovery mechanisms enable you to attach the same metadata to your traces as your metrics.
+For example, for Kubernetes users this means that you can dynamically attach metadata for namespace, pod, and container name of the container sending spans.
+
+```
+tempo:
+  ...
+  scrape_configs:
+  - bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+    job_name: kubernetes-pods
+    kubernetes_sd_configs:
+    - role: pod
+    relabel_configs:
+    - source_labels: [__meta_kubernetes_namespace]
+      target_label: namespace
+    - source_labels: [__meta_kubernetes_pod_name]
+      target_label: pod
+    - source_labels: [__meta_kubernetes_pod_container_name]
+      target_label: container
+    tls_config:
+      ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+      insecure_skip_verify: false
+```
+
+This feature isn’t just useful for Kubernetes users, however.
+All of Prometheus' [various service discovery mechanisms](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#configuration-file) are supported here.
+This means you can use the same scrape_configs between your metrics, logs, and traces to get the same set of labels,
+and easily transition between your observability data when moving from your metrics, logs, and traces.
+
+To configure it, refer to the `scrape_configs` block in the [config reference](https://github.com/grafana/agent/blob/main/docs/configuration-reference.md#tempo_instance_config).
+
+#### Trace discovery through automatic logging
+
+Automatic logging provides an easy and fast way of getting trace discovery through logs.
+Automatic logging writes a well formatted log line to a Loki instance for each span that is processed in the pipeline.
```

Comment:
also supports dumping to stdout

or, frankly, may want to just say "writes well formatted log lines to help with trace discovery" and leave details for the linked doc.